test/functional.test.ts

```diff
@@ -0,0 +1,67 @@
+import _ from 'lodash';
```

Comment:
are these the e2es?

Reply:
Does e2es means End-to-End Solution?

This test uses oak-js-api, neumann chain, polkadot.js api.

I think it should be considered e2e.

Reply:
e2e's means end-to-end test. So it touches multiple integrations and parts of the stack

Reply:
https://www.digitalocean.com/community/tutorials/how-to-write-end-to-end-tests-in-node-js-using-puppeteer-and-jest

Reply:
Okay, I will study it well.